src/UI/components/home/aboutBanner.tsx

```diff
@@ -8,21 +8,26 @@ export default function AboutBanner() {
         justifyContent: 'justify-between',
         alignItems: 'center',
       }}>
-      <Typography variant='h5' color='primary' py='25px' px='35px' sx={{ width: '60%' }}>
-        Analyses-Ready Data and Spatial Models Specifically Tailored to Inform Malaria Vector Control
-      </Typography>
-      <Divider orientation='vertical' flexItem />
-      <Box p='35px'>
-        <Typography mb='25px' variant='body1' textAlign='justify'>
-          The Vector Atlas brings together a new collaboration of partners (icipe, University of Oxford, MAP, PAMCA, GBIF, VectorBase, IRMapper, BMGF) in an initiative to build an online, open access
-          repository to hold and share our analyses-ready malaria vector occurrence, bionomics, abundance, and insecticide resistance data. Our data will be fully up to date and form the basis of a
-          series of spatial models specifically tailored to inform the control of mosquito vectors of disease.
-        </Typography>
-        <Grid container justifyContent='flex-end'>
-          <Button variant='contained'>Join Mailing List</Button>
-          <Button variant='outlined'>Find out more</Button>
+      <Grid container>
+        <Grid item lg={4} md={6} sm={12}>
+          <Typography variant='h4' color='primary' py='25px' px='25px'>
+              Analyses-Ready Data and Spatial Models Specifically Tailored to Inform Malaria Vector Control
+          </Typography>
         </Grid>
-      </Box>
+        <Grid item lg={8} md={6} sm={12}>
+          <Box px='25px' py='25px'>
+            <Typography mb='25px' variant='body1' textAlign='justify'>
+                The Vector Atlas brings together a new collaboration of partners (icipe, University of Oxford, MAP, PAMCA, GBIF, VectorBase, IRMapper, BMGF) in an initiative to build an online, open access
+                repository to hold and share our analyses-ready malaria vector occurrence, bionomics, abundance, and insecticide resistance data. Our data will be fully up to date and form the basis of a
+                series of spatial models specifically tailored to inform the control of mosquito vectors of disease.
+            </Typography>
+            <Grid container justifyContent='flex-end'>
+              <Button variant='contained'>Join Mailing List</Button>
```

Comment:
Maybe we should remove this until we have something set up? I can imagine PAMCA people might try and click on this.